### PR TITLE
miniupnpc: update to 2.0.20161216

### PR DESCRIPTION
**Pull request by mattsm**
Install listdevices as well

Signed-off-by: Mo Chen <uncorrupt@gmail.com>

### CODE REVIEW

File: net/miniupnpc/Makefile [hunk 1]
```diff
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=miniupnpc
-PKG_VERSION:=1.9
+PKG_VERSION:=2.0
```
Comment (diizzyy) [inline net/miniupnpc/Makefile]: Please bump it to the latest snapshot while at it.
miniupnpc-2.0.20161216.tar.gz at the time of writing

File: net/miniupnpc/Makefile [hunk 2]
```diff
@@ -8,12 +8,12 @@
...
 PKG_RELEASE:=1
 
 PKG_SOURCE_URL:=http://miniupnp.free.fr/files
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_MD5SUM:=5ef3ba321e6df72d6519b728b292073e
+PKG_MD5SUM:=2acc4ec912c15447a40cf14ae50df7b9
```
Comment (diizzyy) [inline net/miniupnpc/Makefile]: Change this into PKG_HASH:=<sha256 checksum here>

**Comment by diizzyy** [comment]
Have you compile and run-time-tested it?

**Comment by mattsm** [comment]
Will address the issues. We've not done extensive testing, but we're using parts of it successfully. Any other recommended tests you'd like to see verified?

**Comment by diizzyy** [comment]
I doubt it wouldn't work, the only applications I know that uses miniupnpc are transmission and zerotierone which both are pulling it internally right now.

**Comment by mattsm** [comment]
Well the updates are pushed for when you get a chance to review. Thanks.

**Pull request merged**